lib/shoulda/matchers/active_model/validate_numericality_of_matcher.rb

```diff
@@ -558,7 +558,8 @@ def add_submatcher(submatcher)
           end
 
           if submatcher.respond_to?(:diff_to_compare)
-            @diff_to_compare = [@diff_to_compare, submatcher.diff_to_compare].max
+            @diff_to_compare = [@diff_to_compare,
```

Comment:
Maybe:

``` ruby
@diff_to_compare = [
  @diff_to_compare,
  submatcher.diff_to_compare,
].max
```

Again, the whole "keep indentation to multiple of 2 and don't add more than is necessary" sort of thing